pages/common/podman-login.md

```diff
@@ -0,0 +1,16 @@
+# podman login
+
+> Log in to a container registry.
+> More information: <https://docs.podman.io/en/latest/markdown/podman-login.1.html>.
+
+- Log in to a registry (non-persistent on Linux; persistent on Windows/macOS):
+
+`podman login {{registry.example.org}}`
+
+- Log in to a registry persistently on Linux:
+
+`podman login --authfile=$HOME/.config/containers/auth.json {{registry.example.org}}`
```

Comment:
TBH this is the main reason for adding this page: I can never remember this persistent path with confidence despite being a reasonably experienced Linux administrator at this point.